docs/DafnyRef/Specifications.md

```diff
@@ -497,10 +500,10 @@ are not allowed to modify any memory.
 ## 5.5. Iterator Specification {#sec-iterator-specification}
 ````grammar
 IteratorSpec =
-  { ReadsClause(allowLemma: true, allowLambda: false,
+  { ReadsClause(allowLemma: false, allowLambda: false,
```

Comment:
Since these switched from `true` to `false` (here and above in `LambdaSpec`) I wanted to double check this difference.

Reply:
Resolving per comment below.